requirements.txt

```diff
@@ -23,13 +23,13 @@ matplotlib==3.3.0         # via moonstone (setup.py), scikit-bio
 msgpack==1.0.0            # via cachecontrol
 natsort==7.0.1            # via scikit-bio
 numpy==1.18.1             # via hdmedians, matplotlib, moonstone (setup.py), pandas, patsy, scikit-bio, scikit-learn, scipy, statsmodels
-pandas==1.0.1             # via moonstone (setup.py), scikit-bio, statsmodels
+pandas~=0.25.3
```

Comment:
Did you modify the `requirements.txt` file on purpose? I am not sure we want to downgrades some of the package do we?

Reply:
The pandas downgrade was not on purpose.

Reply:
I restored the v1.0.1 requirement in a commit.

Reply:
Okay ! it seems that there has been other downgrades in the requirements.txt file. In the end, this file is automatically generated using `pip-tools` and the command `pip-compile`, so perhaps you could just drop the changes from the `requirements.txt` file?

Reply:
Yeah, I'll do that. I let PyCharm make a change to add the `filetype` requirement. I did not know that other changes would be made. Once I figure out how I will drop the `requirements.txt` file from the push request.

Reply:
I did a `checkout` of `requirements.txt` to `origin/master` to get back to the original file. 
This will be in the next commit for this branch, then I won't make any modifications.
I thought that I could just drop the file from the pull request altogether, but did not see how to do that.